wagtail/admin/templatetags/wagtailadmin_tags.py

```diff
@@ -1347,23 +1347,23 @@ def keyboard_shortcuts_dialog(context):
                     f"{KEYS.MOD} + {KEYS.SHIFT} + z" if is_mac else f"{KEYS.MOD} + y",
                 ),
             ],
-            ("actions-model", _("Actions")): [
+            ("actions", _("Actions")): [
                 (_("Save changes"), f"{KEYS.MOD} + s"),
                 (_("Preview"), f"{KEYS.MOD} + p"),
-                (_("Toggle sidebar"), "["),
-                (_("Toggle minimap"), "]"),
                 (_("Add or show comments"), f"{KEYS.CTRL} + {KEYS.ALT} + m")
                 if comments_enabled
                 else None,
             ],
-            ("rich-text-content", _("Text content")): [
-                (_("Insert or edit a link"), f"{KEYS.MOD} + k")
+            ("application", _("Application")): [
+                (_("Toggle sidebar"), "["),
+                (_("Toggle minimap"), "]"),
+                (_("Close modal dialogs (like this one)"), f"{KEYS.ESC}"),
             ],
-            ("rich-text-formatting", _("Text formatting")): [
+            ("text-formatting", _("Text formatting")): [
+                (_("Bold"), f"{KEYS.MOD} + b"),
                 (_("Italic"), f"{KEYS.MOD} + i"),
-                (_("Underline"), f"{KEYS.MOD} + u"),
```

Comment:
I'm good with removing this. We probably don't want to encourage underline use, anyway, since it could fool people into thinking it's a link.